kerascv/layers/losses/hard_neg_miner.py

```diff
@@ -0,0 +1,78 @@
+import tensorflow as tf
+
+
+class HardNegativeMiner(tf.keras.layers.Layer):
```

Comment:
"HardNegativeMining" would be more in line with the usual layer naming conventions (e.g. `Masking`)

Reply:
Good point. Done.